src/unstract/sdk/index.py

```diff
@@ -388,7 +384,28 @@ def generate_file_id(
         hashed_index_key = ToolUtils.hash_str(json.dumps(index_key, sort_keys=True))
         return hashed_index_key
 
-    @deprecated("Instantiate Index and call index() instead")
+    @deprecated("Instantiate generate_index_key() instead")
```

Comment:
NIT: @gaya3-zipstack we could provide `version` argument, like [so](https://github.com/Zipstack/unstract-sdk/blob/f8a425af2f00be4f6a19a1bf9e117c311d04cfaa/src/unstract/sdk/tool/stream.py#L234) to keep track of when something was deprecated.

Reply:
That is a great add-on. Done. Thanks @chandrasekharan-zipstack